pyproject.toml

```diff
@@ -69,10 +69,36 @@ version = { source = "file", path = "chowda/_version.py" }
 [tool.pytest.ini_options]
 testpaths = ["tests"]
 
+[tool.ruff.flake8-quotes]
+inline-quotes = "single"
+
 [tool.ruff]
+select = [
+    "B", # flake8-bugbear
+    "C4", # flake8-comprehensions
+    "C90", # mccabe
+    "E", # pycodestyle errors
+    "ERA", # eradicate
+    "F", # pyflakes
+    # "I", # isort
+    "INT", # flake8-gettext
+    "N", # pep8-naming
+    "PIE", # flake8-pie,
+    "PLC", # pylint - convention
+    "PLE", # pylint - error
+    "PLW", # pylint - warning
+    "Q", # flake8-quotes
+    "RET", # flake8-return,
+    "RUF", # Ruff-specific rules
+    "SIM", # flake8-simplify
+    "UP", # pyupgrade
+    "W", # pycodestyle warnings
+
+]
 ignore = [
-    # line too long
-    "E501",
-    # imported but not used
-    "F401"
+    "E501", # line too long
+    "F401", # imported but not used
+    "ERA001", # Commented out code
+    "RET503", # Missing explicit `return`
+
```

Comment:
```
    "F401", # imported but not used
    "ERA001", # Commented out code
    "RET503", # Missing explicit `return`
```
I find these rules pretty helpful. `F401` and `ERA001` keep things tidy, and `RET503` really helps me avoid accidentally returning `None` which I do whenever I forget that Python doesn't return the last line by default like Ruby. Are these rules getting in the way somewhere?

Reply:
I only ignored these while we've been bootstrapping the project. 

- `F401` totally ok removing this now. There's only a few rogue imports to remove.
- `ERA001` Doesn't error. Removing.
- `RET503` The only place this complains is `db.py+23`, where we can `# noqa` it, since the returns are in the protected blocks, and the function raises an `Exception` instead of returning.